src/mod/event_handlers/mod_json_cdr/mod_json_cdr.c

```diff
@@ -305,24 +314,29 @@ static void process_cdr(cdr_data_t *data)
 				close(fd);
 				if (x < 0) {
 					switch_log_printf(SWITCH_CHANNEL_UUID_LOG(data->uuid), SWITCH_LOG_ERROR, "Error writing [%s]\n",path);
+					prometheus_increment_cdr_error();
 					if (0 > unlink(path))
 						switch_log_printf(SWITCH_CHANNEL_UUID_LOG(data->uuid), SWITCH_LOG_ERROR, "Error unlinking [%s]\n",path);
 					backup_cdr(data);
 				} else {
+					prometheus_increment_cdr_success();
 					if(!zstr(data->tmpdir)) {
 						char *move_path = switch_mprintf("%s%s%s", data->logdir, SWITCH_PATH_SEPARATOR, data->filename);
 						if(move_path) {
 							if (rename(path, move_path) == 0) {
 								switch_log_printf(SWITCH_CHANNEL_UUID_LOG(data->uuid), SWITCH_LOG_INFO, "Move CDR [%s] to [%s]\n", data->filename, data->logdir);
+								prometheus_increment_tmpcdr_move_success();
```

Comment:
Increment both cdr success and tmpcdr move success here

Reply:
Done